### ISSUE TITLE
EFOSC Spectra Metadata Analysis: EXPTIME vs LAMRMS

### ISSUE THREAD

**Issue by thespacedoctor** [thespacedoctor/pessto-phase-III-ssdr2-issues#35]
![](http://i.imgur.com/7lA3aJ2.png)

Not sure why the RMS for the dispersion solution is relatively high for the file `SN2013fq_20131015_Gr16_OG530_slit1.0_57001_1.fits`. Probably fine. Any ideas Cosimo?

**Comment by cinserra** [comment]
No, but the wavelength calibration seems off of ~7 Angstrom. If I recall well I rejected it from the RMS according to the Chauvenet's criterion.

**Comment by thespacedoctor** [comment]
so, we're ok to keep this file as it is?

**Comment by smarttgit** [comment]
Cosimo - check this one. If the wavelength solution is OK, we leave it. If not we need to change. e.g. if the wavlength calibration is off by 7 we should change

**Comment by cinserra** [comment]
It is off by 7 Angstrom. Solutions are redo that or apply a rigid shift of 7 Angstrom and report it in the header (if we have to)

**Comment by cinserra** [comment]
OK, it is actually off of 10 Angstrom (too blue of 10 Ang). You have to add a correction of +10 Ang.

**Comment by thespacedoctor** [comment]
done
